components/todoist/sources/common-task.js

```diff
@@ -0,0 +1,32 @@
+const todoist = require("../todoist.app.js");
+const common = require("./common.js");
+
+module.exports = {
+  ...common,
+  props: {
+    todoist,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 5,
+      },
+    },
+    db: "$.service.db",
```

Comment:
These props are also very common and would be a good idea to have them on `sources/common.js`.

The we can do:
```js
module.exports = {
  ...common,
  props: {
    ...common.props,
    selectProjects: { propDefinition: [todoist, "selectProjects"] },
  },
};
```